ledger/src/blockstore_processor.rs

```diff
@@ -1285,6 +1291,17 @@ fn load_frozen_forks(
                 }
 
                 if last_free.elapsed() > Duration::from_secs(10) {
+                    // Purge account state for all dropped banks
+                    for (pruned_slot, pruned_bank_id) in pruned_banks_receiver.try_iter() {
+                        // Simulate this purge being from the AccountsBackgroundService
+                        let is_from_abs = true;
+                        new_root_bank.rc.accounts.purge_slot(
+                            pruned_slot,
+                            pruned_bank_id,
+                            is_from_abs,
+                        );
+                    }
```

Comment:
Nice addition! Do you have a sense for how many pruned slots end up in here?

Reply:
hmmm i would imagine it's `<=` to the number of banks that were dropped from BankForks when the new root was set, depending if any of those were sent to other services

It should be no worse than before because that cleanup was running implicitly on drop anyways